### PR TITLE
Minor variable cleanup

### PR DESCRIPTION
**Pull request by anselmbradford**
`@fcm-bp-max` and `@fcm-bp-min` were removed in https://github.com/cfpb/capital-framework/commit/931149df0c40cf3e01e3b70a003b1acaf6f30206#diff-8db0ab7c481376d02c8ba276e4852714, but remained in the documentation.

## Removals

- Removes unused `@fcm-bp-max` and `@fcm-bp-min` variables.

## Changes

- Updates variables paragraph to apply to components that have a variety of variables, not only color and sizing.
- Alphabetizes variable sections.
- Align variable values per section.
- Remove unneeded top-level section in cf-icons.
- Added missing TOC entry in cf-typography.
- Re-generate yarn.lock and cache.

## Testing

1. Pull branch, run `cd docs && gulp && bundle exec jekyll serve`
2. Visit components on http://127.0.0.1:4000/components/ and see that variables table of contents links work.

### CODE REVIEW

File: packages/cf-icons/usage.md [hunk 1]
```diff
@@ -24,24 +22,26 @@ Framework.
 
 ## SVG icons
```
Comment (Scotchester) [inline packages/cf-icons/usage.md]: We could remove this heading and pull everything else up a level, now that the icon font is gone.

File: packages/cf-typography/usage.md [hunk 2]
```diff
@@ -37,8 +37,50 @@ and has more basic typography patterns.
...
+@meta-header_border:        @gray-40;
+
+// Links
+
+// .a-link__jump
+@jump-link_bg:              @gray-10;
+@jump-link_border:          @gray-40;
+```
 
 ### Font variables
```
Comment (Scotchester) [inline packages/cf-typography/usage.md]: The TOC entry for this needs to be updated. Apparently, it used to be "Font source variables".

File: docs/_includes/usage/cf-layout/usage.md [hunk 3]
```diff
@@ -59,51 +60,55 @@ dependencies of this component.
...
+@hero-knockout-bg:          @gray;
+@hero-knockout-text:        @white;
 
 // Featured Content Module variables
-@fcm-bg:                        @block__bg;
+@fcm-bg:                    @block__bg;
+```
 
-@fcm-bp-max:                    700px;
-@fcm-bp-min:                    701px;
```
Comment (Scotchester) [inline docs/_includes/usage/cf-layout/usage.md]: Also, my bad. I should've removed those!

File: packages/cf-typography/usage.md (hunk 2 above)
Comment (anselmbradford) [reply packages/cf-typography/usage.md]: It has both "Font variables" and "Font source variables"
I added the missing TOC entry, but we may want to combine or nest those.

File: packages/cf-icons/usage.md (hunk 1 above)
Comment (anselmbradford) [reply packages/cf-icons/usage.md]: Done

**Pull request merged**